cellxgene_resources/curation_sample_code.ipynb

```diff
@@ -849,6 +849,23 @@
     "adata.var"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Update some Ensembl IDs based on previous analysis**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "v44_gene_map = json.load(open('../gene_ID_mapping/gene_map_v44.json'))\n",
+    "adata.var.rename(index=v44_gene_map, inplace=True)"
```

Comment:
Jim had put in an extra check to see that the newly mapped ID is not already in the index.

Reply:
Good catch. Please review the added line

Reply:
Looks good!